verify-boilerplate.sh

```diff
@@ -31,8 +31,7 @@ TOOLS="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
 # Directory to check. Default is the parent of the tools themselves.
 ROOT="${1:-${TOOLS}/..}"
 
-boilerDir="${ROOT}/boilerplate"
-boiler="${boilerDir}/boilerplate.py"
+boiler="${TOOLS}/boilerplate/boilerplate.py"
```

Comment:
should we also fix the script to check the return code?

Reply:
Would be better, yes. I don't have a solution readily available, though.